### ISSUE TITLE
SmartCity-Accessibility #307 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#190]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 181 lines...]
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.063 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.navigation.NavigationTest
lat is : 32.078436 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 12.235 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Failed tests: 
  SearchTest.test3:89 null

Tests run: 48, Failures: 1, Errors: 0, Skipped: 5

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:29 min
[INFO] Finished at: 2017-01-19T14:21:02+00:00
[INFO] Final Memory: 29M/199M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] aebace539f4f927f9cd1f320d39305e902358c57 - added spinning wheel and minor refactoring

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/307/)

**Comment by kapalex** [comment]
@Kolikant Searchtest failed, I don't think I changed anything related to it
Can you take a look?

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 154 lines...]
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.984 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.navigation.NavigationTest
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078436 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 11.979 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Failed tests: 
  SearchTest.test3:89 null

Tests run: 46, Failures: 1, Errors: 0, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:24 min
[INFO] Finished at: 2017-01-19T14:28:11+00:00
[INFO] Final Memory: 28M/203M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] aebace539f4f927f9cd1f320d39305e902358c57 - added spinning wheel and minor refactoring

- [Alexander Kaplan] 57c07d6ea45905b3ea3353144828006d3917b470 - remove deprecated classes



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/308/)

**Comment by Kolikant** [comment]
When i get home, :)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
